### ISSUE TITLE
Unable to parse replays

### ISSUE THREAD

**Issue by jordanghub** [nickbabcock/rrrocket#265]
Hello
I'm getting this error when trying to parse a replay since the season 16 update (patch 2.43)

```
$ ./rrrocket.exe 58B229AC4AD87D13BF51EFAB18A8659B.replay -n
An error occurred: Unable to parse replay 58B229AC4AD87D13BF51EFAB18A8659B.replay

Caused by:
    0: Error decoding frame: attribute unknown or not implemented: actor id / actor object id / attribute id: 3 / 127 / 21. found attribute TAGame.GameEvent_TA:MatchStartEpoch (unknown to boxcars) on Archetypes.GameEvent.GameEvent_Soccar in network cache data. This is likely due to a rocket league update or an atypical replay. File a bug report! Context: on frame: 0, last updated actor: (actor stream id / object id / name: 2 / 81 / Archetypes.CarComponents.CarComponent_Dodge, attribute stream id / object id / name: 21 / 23 / TAGame.CarComponent_TA:Vehicle, attribute: ActiveActor(ActiveActor { active: true, actor: ActorId(9) }))
    1: Error decoding frame: attribute unknown or not implemented: actor id / actor object id / attribute id: 3 / 127 / 21. found attribute TAGame.GameEvent_TA:MatchStartEpoch (unknown to boxcars) on Archetypes.GameEvent.GameEvent_Soccar in network cache data. This is likely due to a rocket league update or an atypical replay. File a bug report! Context: on frame: 0, last updated actor: (actor stream id / object id / name: 2 / 81 / Archetypes.CarComponents.CarComponent_Dodge, attribute stream id / object id / name: 21 / 23 / TAGame.CarComponent_TA:Vehicle, attribute: ActiveActor(ActiveActor { active: true, actor: ActorId(9) }))
    2: attribute unknown or not implemented: actor id / actor object id / attribute id: 3 / 127 / 21
```
[58B229AC4AD87D13BF51EFAB18A8659B.zip](https://github.com/user-attachments/files/16928011/58B229AC4AD87D13BF51EFAB18A8659B.zip)

**Comment by nickbabcock** [comment]
Thanks for the report, I'll push out a release tonight.

**Comment by nickbabcock** [comment]
Released as part of [0.9.12](https://github.com/nickbabcock/rrrocket/releases/tag/v0.9.12)
